pkg/config/env/environment_container_features.go

```diff
@@ -23,6 +23,8 @@ const (
 	EKSFargate Feature = "eksfargate"
 	// KubeOrchestratorExplorer can be enabled
 	KubeOrchestratorExplorer Feature = "orchestratorexplorer"
+	// ECSOrchestratorExplorer can be enabled
+	ECSOrchestratorExplorer Feature = "ecsorchestratorexplorer"
```

Comment:
Is it intended to not have non alphanumeric characters here or can we have something clearer like `orchestratorexplorer_ecs` (and maybe in near future rename to `orchestratorexplorer_k8s`)?

Reply:
https://github.com/DataDog/datadog-agent/blob/7225ef1ca869e403d455ac5c66e3f3086a0c3fb3/pkg/config/env/environment_container_features.go#L25-L28